src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/osd/osd-list/osd-list.component.html

```diff
@@ -100,3 +100,15 @@
     </ng-container>
   </ng-container>
 </ng-template>
+
+<ng-template #osdIdTpl
+             let-row="row"
+             let-value="value">
+  <i [ngClass]="[icons.spinner, icons.spin]"
+     *ngIf="row.cdExecuting"></i>
+  <strong>
+    {{ value }}
+  </strong>
+  <span *ngIf="row.cdExecuting"
+        class="text-muted italic">({{ row.cdExecuting }})</span>
+</ng-template>
```

Comment:
Instead of creating a new template just for osd, maybe we can update the one in TableComponent to accept a class.
Something like `cdExecutingClass`
```suggestion
```

Reply:
I use the `customTemplateConfig`, lease see https://github.com/ceph/ceph/pull/35039/commits/2d4f1fc46509153f546b29acb28e970bf720433c

```
    this.columns = [
      {
        prop: 'id',
        name: $localize`ID`,
        flexGrow: 1,
        cellTransformation: CellTemplate.executing,
        customTemplateConfig: {
          valueClass: 'a',
          executingClass: 'b'
        }
      },
```

Which makes the cell value render with class `a` and the executing state
(e.g. (updating)) render with class `b`.